portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,62 @@
+*,
+*::before,
+*::after{
+  box-sizing:border-box;
+}
+:root {
+  --white-colour: white;
```

Comment:
is white-colour var used anywhere?

Reply:
I was supposed to use it on line 56, but I must've forgotten, I will fix it